ci/jobs/scripts/openstack-upgrade-pulp.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+echo "Start upgrading Pulp on ${INSTANCE_IP}..."
+alias SSH_CLI="ssh -t -o StrictHostKeyChecking=no cloud-user@${INSTANCE_IP}"
```

Comment:
Style nit, but I'm not used to seeing UPPER_CASE aliases; these look like envvars to me. Since ssh itself isn't used later on in the script `alias ssh="ssh -t ...` might be good enough.